### ISSUE TITLE
ignore files copied to bundle from electron dist

### ISSUE THREAD

**Issue by Kaffiend** [Cedware/electron-angular-toolkit#10]
support glob pattern for ignored files in the build process?

Swapped electron to typescript, would be nice to ignore .ts files or any other files users dont want output to  build dir

**Comment by Cedware** [comment]
Actually there shouldn't be any .ts files inside your output directory, can you provide me an example?

**Comment by Kaffiend** [comment]
Yeah, ill throw one together. I switched the electron main and other parts
for the main process over to typescript and added a build script.

**Comment by Kaffiend** [comment]
here is an example
https://github.com/Kaffiend/electron-ts-example
